posthog/models/app_metrics/sql.py

```diff
@@ -64,9 +65,24 @@
     error_details String CODEC(ZSTD(3))
 )
 ENGINE={kafka_engine(topic=KAFKA_APP_METRICS)}
+SETTINGS kafka_handle_error_mode='stream'
 """
 )
 
+# MergeTreeEngine(table_name, replication_scheme=ReplicationScheme.REPLICATED)
+KAFKA_APP_METRICS_DLQ_SQL = lambda: KAFKA_ENGINE_DLQ_BASE_SQL.format(
+    table="kafka_app_metrics_dlq",
```

Comment:
Nitty suggestion: I'm not sure about this, but how would you feel about naming this `kafka_dlq_app_metrics` and updating others as well. Reasoning: We have both a prefix and suffix here, which is IMO confusing.